### PR TITLE
Add function to get createdAt field for diary labels  

### PR DESCRIPTION
**Pull request by dkeden**
This PR adds a function to request the `createdAt` field (from the API) for a given `patient_id`. The `createdAt` field indicates when a diary was created (i.e. when the user signed up).

I have also added the `createdAt` field in the `get_diary_labels` query string, and added a line for the dataframe. It may be redundant to add in both places, but figured that in cases when just the `createdAt` is required, it would be unnecessary to request all diary labels.

New:
seerpy: `get_diary_created_at()`
graphql: `get_diary_created_at_query_string()`

Changes:
seerpy: `get_diary_labels_dataframe()` --> added line
graphql: `get_diary_labels_query_string()` --> added line

### CODE REVIEW

File: seerpy/graphql.py [hunk 1]
```diff
@@ -343,7 +352,8 @@ def get_diary_labels_query_string(patient_id, label_type, limit, offset, from_ti
                 id
                 diary {
                     id
-                    labelGroups (filters: [{name: "labelType", value:"%s"}]) {
+                    createdAt
+                    labelGroups {
```
Comment (pkaroly) [inline seerpy/graphql.py:356]: no filter?

Comment (dkeden) [reply seerpy/graphql.py:356]: Oh man, I must've accepted the wrong change due to conflict. Thanks for picking up on this

Comment (dkeden) [reply seerpy/graphql.py:356]: Hey, sorry. Hopefully that should be fixed up

Comment (bjdoyle) [reply seerpy/graphql.py:356]: I don't see it?

Comment (dkeden) [reply seerpy/graphql.py]: Now?

Comment (dkeden) [reply seerpy/graphql.py]: i've had some really weird issues w/ github today

Comment (bjdoyle) [reply seerpy/graphql.py]: yep all good now. yeah me too, it seems to be struggling. jumping back and forth between multiple PRs also gets me confused!

Comment (dkeden) [reply seerpy/graphql.py]: I agree. It was difficult over the break to split these PRs up, but require all of them for dev

Comment (dkeden) [reply seerpy/graphql.py]: I'll try putting the next lot into one PR - a bunch of small changes anyway

**Pull request merged**